src/Contracts/SelfServeContracts.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Messaging types used with SelfServe Component <-> Portal communication
+ * and Hosted <-> SelfServe Component communication
+ */
+
+export enum SelfServeMessageTypes {
+  TelemetryInfo = 0,
```

Comment:
Can these be made into string enums on the portal side? The numeric enums are a huge pain to debug and there is zero perf benefit.